vespalog/src/main/java/com/yahoo/log/impl/InitializeLog.java

```diff
@@ -0,0 +1,21 @@
+package com.yahoo.log.impl;
+
+import com.yahoo.log.LogSetup;
+
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Sets up Vespa logging. Call a setup method to set up this.
+ *
+ * @author baldersheim
+ */
+public class InitializeLog {
+    private static final Logger log = Logger.getLogger(InitializeLog.class.getName());
+
+    public static void init(String programName) {
+        LogSetup.initVespaLogging(programName);
+        log.log(Level.INFO, "Initialized logging for " + programName);
```

Comment:
When sifting through large amounts of logs it is often useful to know when a process started.  That was the intention of the `Starting jdisc at version 7.581.34` message.  If the initialization of the logging is moved so should that message (and this message is redundant).

Reply:
This is just a draft, not ready for review yet :)

Reply:
The log message you mention could be moved to ApplicationLoader, but without outputting the version which is not available for jdisc_core. Making it available is outside the scope of this PR.

Reply:
Not sure what's best.